newIDE/app/src/ExportAndShare/ShareDialog/index.js

```diff
@@ -116,14 +116,19 @@ const ShareDialog = ({
   const [currentTab, setCurrentTab] = React.useState<ShareTab>(
     initialTab || getShareDialogDefaultTab()
   );
+  const showOnlineWebExporterOnly = !automatedExporters && !manualExporters;
   const [
     chosenExporterSection,
     setChosenExporterSection,
-  ] = React.useState<?ExporterSection>(null);
+  ] = React.useState<?ExporterSection>(
+    showOnlineWebExporterOnly ? 'browser' : null
+  );
   const [
     chosenExporterSubSection,
     setChosenExporterSubSection,
-  ] = React.useState<?ExporterSubSection>(null);
+  ] = React.useState<?ExporterSubSection>(
+    showOnlineWebExporterOnly ? 'online' : null
+  );
```

Comment:
This is the fix. But the change in the other file ensures that even if we changes this to be `null` by default (i.e: we don't force a choice), then we still have the list of choices shown (with just gd.games - would be weird, but it would work and not shown nothing).